core/controllers/editor_test.py

```diff
@@ -697,6 +702,73 @@ def test_deletion_rights_for_published_exploration(self):
         self.assertEqual(response.status_int, 200)
         self.logout()
 
+    def test_logging_info_after_deletion(self):
+        """Tests correctness of logged statements while deleting exploration"""
+        logging_info = []
+
+        def add_logging_info(msg, *_):
+            if msg != 'all_pending: clear %s':
+                logging_info.append(msg)
+
+        with self.swap(logging, 'info', add_logging_info), self.swap(
+            logging, 'debug', add_logging_info):
+            # checking for normal user
```

Comment:
Checking for non-moderator/admin.

(Here and elsewhere: start comments with capital letters and end with full stops.)